src/main/java/metrics_influxdb/InfluxdbUdp.java

```diff
@@ -72,4 +80,11 @@ public int sendRequest(boolean throwExc, boolean printJson) throws Exception {
 
 		return 0;
 	}
+
+	private JsonBuilder getJsonBuilder(){
+		if("0.9".equals(version)){
+			return new JsonBuilderV9(this.database, "");
+		}
+		return  new JsonBuilderDefault();
```

Comment:
Again: Does this really match every other version, including future ones? What about version 0.9.1 or version 1.0?